lightning_examples/finetuning-scheduler/finetuning-scheduler.py

```diff
@@ -376,25 +367,22 @@ def __init__(
         self.no_decay = ["bias", "LayerNorm.weight"]
 
     @property
-    def finetuningscheduler_callback(self) -> FinetuningScheduler:  # type: ignore # noqa
-        fts = [c for c in self.trainer.callbacks if isinstance(c, FinetuningScheduler)]  # type: ignore # noqa
-        return fts[0] if fts else None
+    def finetuningscheduler_callback(self) -> fts.FinetuningScheduler:
+        fts_callback = [c for c in self.trainer.callbacks if isinstance(c, fts.FinetuningScheduler)]
+        return fts_callback[0] if fts_callback else None
 
     def forward(self, **inputs):
         return self.model(**inputs)
 
     def training_step(self, batch, batch_idx):
         outputs = self(**batch)
         loss = outputs[0]
-        self.log("train_loss", loss)
+        self.log("train_loss", loss, sync_dist=True)
```

Comment:
this won't have any effect unless you do `on_epoch=True` in `training_step` which is `False` by default here.

Reply:
good point, thanks! I over-zealously applied these ``sync_dist``s as a workaround when I was debugging a separate multi-gpu ddp_spawn issue but it is not relevant in this case and it definitely shouldn't be included in a tutorial!

Reply:
I'll push another commit shortly with the ``sync_dist``s removed